dev-util/bindgen/metadata.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+    <maintainer type="person">
+        <email>bertrand@jacquin.bzh</email>
+        <name>Bertrand Jacquin</name>
+    </maintainer>
+    <upstream>
```

Comment:
You need to add `proxied="yes"` and the `proxy-maint` project.  See [here](https://wiki.gentoo.org/wiki/Project:Proxy_Maintainers/User_Guide#Proxied_maintainer_in_metadata.xml)

Reply:
I removed myself from the list and instead added rust team